lib/presentation/edit_workout/ParamContainer.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:dataclass/dataclass.dart';
+
+@dataClass
+class ParamContainer {
+  final int _newWeight;
+  final int _newRepCount;
+  final String _workoutId;
+  final String _workoutExerciseId;
+
+  ParamContainer(this._newWeight, this._newRepCount, this._workoutId,
```

Comment:
Can we switch to named args here? It'll make construction a bit clearer in terms of which values are used for what